test/activities/content/ContentHtmlFileEntity.js

```diff
@@ -0,0 +1,104 @@
+/* global fetchMock */
+import { ContentHtmlFileEntity } from '../../../src/activities/content/ContentHtmlFileEntity.js';
+import { contentHtmlFileData } from './data/TestContentHtmlFileEntity.js';
+import { getFormData } from '../../utility/test-helpers.js';
+
+describe('ContentHtmlFileEntity', () => {
+	let htmlFileData;
+	let contentHtmlFileEntity;
+
+	beforeEach(() => {
+		htmlFileData = window.D2L.Hypermedia.Siren.Parse(contentHtmlFileData);
+		contentHtmlFileEntity = new ContentHtmlFileEntity(htmlFileData);
+	});
+
+	afterEach(() => {
+		fetchMock.reset();
+	});
+
+	describe('Reads properties', () => {
+		it('reads title', () => {
+			expect(contentHtmlFileEntity.title()).to.equal('Test Html File Title');
+		});
+
+		it('reads rich text description', () => {
+			expect(contentHtmlFileEntity.descriptionRichText()).to.equal('<p>description text</p>');
+		});
+
+		it('reads text description', () => {
+			expect(contentHtmlFileEntity.descriptionText()).to.equal('description text');
+		});
+	});
+
+	describe('Equality tests', () => {
+		it('Equality should return true when details match', () => {
+			const htmlFileData = {
+				title: 'Test Html File Title',
+				descriptionRichText: '<p>description text</p>'
+			};
+			expect(contentHtmlFileEntity.equals(htmlFileData)).to.equal(true);
+		});
+
+		it('Equality should return false when title is different', () => {
+			const htmlFileData = {
+				title: 'New Title',
+				descriptionRichText: '<p>description text</p>'
+			};
+			expect(contentHtmlFileEntity.equals(htmlFileData)).to.equal(false);
+		});
+
+		it('Equality should return true when descriptions match', () => {

```

Comment:
I left this in here to discuss something I noticed. We have descriptions (plain and rich) for each of our activities ("text" and "html"), although to my knowledge we don't use them. Right now, it looks like these descriptions are not included in the `equals()` methods, but we test for them (like this one). Ex: for LTI links, [we have the equals method](https://search.d2l.dev/xref/BrightspaceHypermediaComponents/siren-sdk/src/activities/content/ContentLTILinkEntity.js?r=7bc5a317#128) that does not include the description, but [we test for it here](https://search.d2l.dev/xref/BrightspaceHypermediaComponents/siren-sdk/test/activities/content/ContentLTILinkEntity.js?r=c62d75d3#62). I'm inclined to just delete these tests since we aren't using the descriptions, but I'm wondering what y'all think.

Reply:
On the one hand it's a little weird that two things can equal while having different descriptions, so maybe it's better to update the equals method, however I agree that as it is now the tests aren't good and if we don't update the equals method I think it makes sense to delete them

Reply:
I agree with you. I will update the equals methods to include both descriptions.

Reply:
Looking into this a bit further, this change is a bit more involved since we would also need to change [the corresponding method in activities](https://search.d2l.dev/xref/BrightspaceHypermediaComponents/activities/components/d2l-activity-editor/d2l-activity-content-editor/lti-link/state/content-lti-link.js?r=32a193d5#59) for each activity. Since we are keeping track of what we are using anyways, I'm leaning towards removing the descriptions tests, and we can add them if we decide to include the descriptions in the future.